webclient/src/forms/KnownHostForm/KnownHostForm.tsx

```diff
@@ -33,6 +33,14 @@ const KnownHostForm = ({ host, onRemove, onSubmit }) => {
     }
   };
 
+  const handleOnSubmit = ({ name, host, port, ...values }) => {
+    name = name.trim();
+    host = host.trim();
+    port = port.trim();
```

Comment:
Not a huge deal obviously, but could the `port` input be set to `type="number"` to remove the need for a trim here?

Reply:
It is currently set to `number`, i wasnt sure if that input type would always protect against the need for trim, so i put this here as a failsafe. Do you think it's unnecessary?